### PR TITLE
Add toast modifier to alert component

### PR DESCRIPTION
**Pull request by NeOMakinG**
Related with https://github.com/PrestaShop/PrestaShop/issues/16595

This PR also contains https://github.com/PrestaShop/PrestaShop/issues/16606 and https://github.com/PrestaShop/PrestaShop/issues/16607 modifications

Job is done but https://github.com/PrestaShop/prestashop-ui-kit/pull/73 needs to be merged before so we can lint

How to test : build assets and compare index.html alerts with invision screens on related PRs

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -766,13 +760,7 @@ <h1>Alerts</h1>
...
             </div>
 
-            <div class="alert expandable-alert alert-danger" role="alert">
-                <button type="button" class="close" data-dismiss="alert" aria-label="Close">
-                    <span aria-hidden="true"><i class="material-icons">close</i></span>
-                </button>
-                <button type="button" class="read-more btn-link" data-toggle="collapse" data-target="#collapseDanger" aria-expanded="false" aria-controls="collapseDanger">
-                    Read more
-                </button>
+            <div class="alert expandable-alert alert-warning" role="alert">
```
Comment (eternoendless) [inline index.html]: I think we should keep the `alert-danger` style in CSS for backwards compatibility

Comment (NeOMakinG) [reply index.html]: What do you mean ? We got it but it's refactored, see :

image

Comment (PierreRambaud) [reply index.html]: @eternoendless https://github.com/PrestaShop/prestashop-ui-kit/pull/75/files#diff-eacf331f0ffc35d4b482f1d15a887d3bL761

Comment (eternoendless) [reply index.html]: My mistake, I thought it was gone.

**Comment by eternoendless** [comment]
Thank you @NeOMakinG

**Pull request merged**